terraform/modules/neptune-cluster-parameter-group/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable "app" {
+  type        = string
+  description = "the name of the application expressed as an acronym"
+  sensitive   = false
+}
+
+variable "env" {
+  type        = string
+  description = "the target tier ('dev', 'qa', 'stage', 'nonprod' or 'prod'.)"
+  sensitive   = false
+
+  validation {
+    condition     = contains(["dev", "qa", "stage", "prod", "nonprod"], var.env)
+    error_message = "valid values are 'dev', 'qa', 'stage', 'prod', and 'nonprod'"
+  }
+}
+
+variable "program" {
+  type        = string
+  description = "the program associated with the application"
+  sensitive   = false
+
+  validation {
+    condition     = contains(["crdc", "ccdi", "ctos", "fnl"], var.program)
+    error_message = "valid values for program are 'crdc', 'ccdi', 'fnl' and 'ctos'"
+  }
+}
```

Comment:
In the datacommons-devops repository, it looks like you want to use the "resource_prefix" variable, instead of providing program, env, and app as separate variables that are combined to create the stack name (which looks similar to resource_prefix). If that's the case, this can be removed.